### PR TITLE
Toggle dev preservation worker deployment

### PR DESCRIPTION
**Pull request by djjuhasz**
Add an $ENDURO_PRES_SYSTEM Tilt environment variable to toggle the
deployment of Enduro with an a3m or Archivematica preservation worker.

### CODE REVIEW

File: Tiltfile [hunk 1]
```diff
@@ -3,22 +3,31 @@ secret_settings(disable_scrub=True)
 load("ext://uibutton", "cmd_button", "text_input")
 load('ext://dotenv', 'dotenv')
 
+# Set preservation system (default: 'a3m')
+PRES_SYS = os.environ.get('ENDURO_PRES_SYSTEM', 'a3m')
+os.putenv('ENDURO_PRES_SYSTEM', PRES_SYS)
```
Comment (sevein) [inline Tiltfile]: Do we need this line (os.putenv)?

Comment (jraddaoui) [reply Tiltfile]: I think just for this: https://github.com/artefactual-sdps/enduro/pull/774/files#diff-c2ee8653e1d6b85f0aadf87cd438a9250806c052877248442be4d434cbc52425R134

Comment (sevein) [reply Tiltfile]: Oh I see, it's used in L134.

**Review by sevein — APPROVED** [review]
LGTM

File: Tiltfile (hunk 1 above)
Comment (djjuhasz) [reply Tiltfile]: Yeah, it's used in L134 but it's pretty hacky. :(  I realized this morning that Starlark can probably do standard Python variable expansion in a string, which would be much better than relying on env var expansion.

**Pull request merged**